src/ScottPlot5/ScottPlot5/FontService.cs

```diff
@@ -1,26 +1,27 @@
 ﻿namespace ScottPlot;
 
 /// <summary>
-/// This class provides cross-platform tools for working with fonts
+/// Cross-platform tools for working with fonts
 /// </summary>
 public static class FontService
 {
-    public static SKTypeface SansTypeface { get; private set; } = GetDefaultSansFont();
-    public static SKTypeface MonospaceTypeface { get; private set; } = GetDefaultMonospaceFont();
-    public static SKTypeface SerifTypeface { get; private set; } = GetDefaultSerifFont();
+    /// <summary>
+    /// This default font is used for almost all text rendering
+    /// </summary>
+    public static string DefaultFontName { get; private set; } = GetDefaultSansFont().FamilyName;
 
-    public static string SansFontName => SansTypeface.FamilyName;
-    public static string MonospaceFontName => MonospaceTypeface.FamilyName;
-    public static string SerifFontName => SerifTypeface.FamilyName;
+    public static string SansFontName { get; private set; } = GetDefaultSansFont().FamilyName;
+    public static string SerifFontName { get; private set; } = GetDefaultSerifFont().FamilyName;
+    public static string MonospaceFontName { get; private set; } = GetDefaultMonospaceFont().FamilyName;
 
-    public static string[] GetInstalledFonts()
+    private static HashSet<string> GetInstalledFonts()
     {
-        return SKFontManager.Default.FontFamilies.ToArray();
+        return new(SKFontManager.Default.FontFamilies, StringComparer.InvariantCultureIgnoreCase);
     }
 
-    private static SKTypeface GetDefaultSansFont()
+    public static SKTypeface GetDefaultSansFont()
     {
-        HashSet<string> installedFonts = new(SKFontManager.Default.FontFamilies);
+        var installedFonts = GetInstalledFonts();
 
         string[] preferredFonts = { "Open Sans", "Segoe UI", "Lato", "DejaVu Sans", "Helvetica" };
```

Comment:
I think users should probably be able to just edit this, one might want to modify the default _and_ set a fallback font, and that's inconvenient to do with only `SetDefaultFont`

Reply:
Thanks for pointing this out! I'll try to simplify this system by making all these methods private (solving the issue) and only exposing a few strings which the user can set how they want if they don't like the defaults:

https://github.com/ScottPlot/ScottPlot/blob/418665bb0e6ed053a29ee53fbc4b0fd22218d2d8/src/ScottPlot5/ScottPlot5/FontService.cs#L11-L15

I don't think we even need a fallback. If the font doesn't exist, SkiaSharp renders it using whatever the system default is